widgets/bat_openbsd.lua

```diff
@@ -19,67 +19,71 @@
 
 -- {{{ Grab environment
 local tonumber = tonumber
-local math = { floor = math.floor, modf = math.modf }
+local math = {
+    floor = math.floor,
+    modf = math.modf
+}
 
-local helpers = require"vicious.helpers"
-local spawn = require"vicious.spawn"
+local helpers = require("vicious.helpers")
 -- }}}
 
-local STATES = { [0] = "↯",         -- not charging
-                 [1] = "-",         -- discharging
-                 [2] = "!",         -- critical
-                 [3] = "+",         -- charging
-                 [4] = "N/A",       -- unknown status
-                 [255] = "N/A" }    -- unimplemented by the driver
+local bat_openbsd = {}
+function bat_openbsd.async(format, warg, callback)
+    local battery_id = warg or "bat0"
```

Comment:
Just realized that this is much better than overriding `warg` :wink:

Reply:
Awesome! :)